swan-cern/files/swan_config_cern.py

```diff
@@ -42,6 +45,18 @@ async def get_swan_user_pod(self):
 
         return self.pod
 
+    def _check_tn_access(self):
+        """
+        Helper function to check if this SWAN deployment is exposed to the Technical Network
+        and this SWAN deployment is exposed to the Technical Network
+        """
+        user_roles = self.spawner.user_roles
+        tn_enabled = get_config('hub.config.SpawnHandlersConfigs.tn_enabled', False)
+        ats_role = get_config('hub.config.SpawnHandlersConfigs.ats_role', "")
```

Comment:
In this function you do inconsistent use of single and double quotes for string literals. Please stick to one.